extensions/sql-migration/src/dashboard/dashboardTab.ts

```diff
@@ -0,0 +1,791 @@
+				preRequisiteListElement,
+				preRequisiteLearnMoreLink])
+			.withLayout({ flexFlow: 'column' })
+			.component();
+
+		tasksContainer.addItem(migrateButton, {});
+		tasksContainer.addItems(
+			[preReqContainer],
+			{ CSSStyles: { 'margin-left': '20px' } });
+
+		return tasksContainer;
+	}
+
+	private _createTaskButton(view: azdata.ModelView, taskMetaData: IActionMetadata): azdata.Component {
+		const maxHeight: number = 84;
+		const maxWidth: number = 236;
+		const buttonContainer = view.modelBuilder.button().withProps({
+			buttonType: azdata.ButtonType.Informational,
+			description: taskMetaData.description,
+			height: maxHeight,
+			iconHeight: 32,
+			iconPath: taskMetaData.iconPath,
+			iconWidth: 32,
+			label: taskMetaData.title,
+			title: taskMetaData.title,
+			width: maxWidth,
+			CSSStyles: {
+				'border': '1px solid',
+				'display': 'flex',
+				'flex-direction': 'column',
+				'justify-content': 'flex-start',
+				'border-radius': '4px',
+				'transition': 'all .5s ease',
+			}
+		}).component();
+		this.disposables.push(
+			buttonContainer.onDidClick(async () => {
+				if (taskMetaData.command) {
+					await vscode.commands.executeCommand(taskMetaData.command);
+				}
+			}));
+		return view.modelBuilder.divContainer().withItems([buttonContainer]).component();
+	}
+
+	private async _createFooter(view: azdata.ModelView): Promise<azdata.Component> {
+		const footerContainer = view.modelBuilder.flexContainer().withLayout({
+			flexFlow: 'row',
+			width: maxWidth,
+			justifyContent: 'flex-start'
+		}).component();
+		const statusContainer = await this._createMigrationStatusContainer(view);
+		const videoLinksContainer = this._createVideoLinks(view);
+		footerContainer.addItem(statusContainer);
+		footerContainer.addItem(
+			videoLinksContainer,
+			{ CSSStyles: { 'padding-left': '8px', } });
+
+		return footerContainer;
+	}
+
+	private _createVideoLinks(view: azdata.ModelView): azdata.Component {
+		const linksContainer = view.modelBuilder.flexContainer()
+			.withLayout({
+				flexFlow: 'column',
+				width: '440px',
+				height: '365px',
+				justifyContent: 'flex-start',
+			}).withProps({
+				CSSStyles: {
+					'border': '1px solid rgba(0, 0, 0, 0.1)',
+					'padding': '10px',
+					'overflow': 'scroll',
+				}
+			}).component();
+
+		const titleComponent = view.modelBuilder.text()
+			.withProps({
+				value: loc.HELP_TITLE,
+				CSSStyles: { ...styles.SECTION_HEADER_CSS }
+			})
+			.component();
+
+		linksContainer.addItems(
+			[titleComponent],
+			{ CSSStyles: { 'margin-bottom': '16px' } });
+
+		const links = [
+			{
+				title: localize('sql.migration.dashboard.help.link.migrateUsingADS', 'Migrate databases using Azure Data Studio'),
+				description: localize('sql.migration.dashboard.help.description.migrateUsingADS', 'The Azure SQL Migration extension for Azure Data Studio provides capabilities to assess, get right-sized Azure recommendations and migrate SQL Server databases to Azure.'),
```

Comment:
Can all these strings live in `strings.ts`, or some other resource file-esque store?

Reply:
Moreover, should we turn these links into aka.ms shortlinks, so 1) we can change what they point to without having to change them in the product and also 2) track link hits? The PMs wanted us to do something similar when we introduced these links into DMA so maybe we can check if they want to do something similar here

Reply:
I consolidated the strings as suggested, let's create a task to consolidate the URL's as suggested and convert them to fw/akams links